src/main/java/com/test/contract/ui/MainController.java

```diff
@@ -70,6 +70,10 @@ public void init() {
         TableColumn<Contact, String> emailColumn = new TableColumn<>("E-mail");
         emailColumn.setCellValueFactory(new PropertyValueFactory<>("email"));
 
+
+        TableColumn<Contact, String> ageColumn = new TableColumn<>("Возраст");
+        ageColumn.setCellValueFactory(new PropertyValueFactory<>("age"));
+
```

Comment:
be more attentive, you have added aditional column >>> ageColumn >> but I don't see this parameter below (in the next line)
>>> table.getColumns().setAll(idColumn, nameColumn, phoneColumn, emailColumn); 

you don't use this additional property(ageColumn) on ui side, please check this point again; 
tnx